src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -122,8 +123,9 @@ private static VersionInfo GetVersionInfo()
             return new VersionInfo(
                 modelSignature: "HASHTRNS",
                 // verWrittenCur: 0x00010001, // Initial
-                verWrittenCur: 0x00010002, // Invert hash key values, hash fix
-                verReadableCur: 0x00010002,
+                //verWrittenCur: 0x00010002, // Invert hash key values, hash fix
+                verWrittenCur: 0x00010003,
```

Comment:
>0x00010003 [](start = 31, length = 10)

Please add a comment explaining why the version is changing (similar to the line above).